back/Gemfile

```diff
@@ -163,6 +163,7 @@ gem 'mjml-rails', '~> 4.9'
 gem 'cloudfront-rails', '~> 0.4'
 gem 'google-cloud-document_ai'
 gem 'combine_pdf'
+gem 'concurrent-ruby', '~> 1.2.3'
```

Comment:
Will move it.

The reasons why I thought the main app was the right place is that we also considered using this gem for tenant templates at some point, it might be useful in other places at some point, and it's possible that we're already using it somewhere else (because the gem was already included through the inclusion of other gems).